src/conways.rs

```diff
@@ -79,6 +92,16 @@ impl Conways {
         }
         count
     }
+
+    pub fn update_state_cell(&mut self, (x, y): Position) {
```

Comment:
This function could have a better name maybe `switch_state_cell` or `toggle_state_cell`